rconweb/api/audit_log.py

```diff
@@ -91,7 +91,7 @@ def get_audit_logs_autocomplete(request):
 @csrf_exempt
 @login_required()
 @permission_required("api.can_view_audit_logs", raise_exception=True)
-@require_http_methods(['POST'])
+@require_http_methods(['GET'])
 @require_content_type()
```

Comment:
when GET you should also remove the require_content_type decorator. GET requests should not have or rely on the body.

Reply:
Whoops, thanks dude. Fixed in #411